engine/frontend/src/component/PackageCatalogForm.js

```diff
@@ -101,6 +101,20 @@ const checkValidIntType = (data) => {
     }
 }
 
+const checkValidFloatType = (data) => {
+    const  isValidFloat = (value) => {
+        return (/^-?[\d]*(\.[\d]+)?$/g).test(value);
+    }
+    if (data === "undefined") {
+        return false
+    }
+    return isValidFloat(data)
+}
+
+const checkValidBooleanType = (data) => {
+    return ["TRUE", "FALSE"].contains(data)
```

Comment:
@Peeeekay I'd recommend lowercasing the data to make sure you can match invariantly to casing